Ports/curl/package.sh

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env -S bash ../.port_include.sh
 port=curl
-version=7.86.0
+version=7.87.0
 useconfigure=true
-files="https://curl.se/download/curl-${version}.tar.bz2 curl-${version}.tar.bz2 f5ca69db03eea17fa8705bdfb1a9f58d76a46c9010518109bb38f313137e0a28"
+files="https://curl.se/download/curl-${version}.tar.bz2 curl-${version}.tar.bz2 5d6e128761b7110946d1276aff6f0f266f2b726f5e619f7e0a057a474155f307"
 auth_type=sha256
 depends=("ca-certificates" "openssl" "zlib" "zstd")
```

Comment:
Have you investigated what's breaking CA certificate loading after one of the recent version bumps?

Reply:
It seems like certificate loading still works for me, so it might just be a setting that we need to hardcode in the CMake configuration.

Reply:
This should work:

```diff
diff --git a/Ports/curl/package.sh b/Ports/curl/package.sh
index 2e37c10e52..64b0c066af 100755
--- a/Ports/curl/package.sh
+++ b/Ports/curl/package.sh
@@ -15,6 +15,8 @@ configure() {
     "${configopts[@]}" \
     -DCURL_USE_OPENSSL=ON \
     -DCURL_ZSTD=ON \
+    -DCURL_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt \
+    -DCURL_CA_PATH=none \
     -DCURL_DISABLE_NTLM=ON \
     -DCURL_DISABLE_SOCKETPAIR=ON \
     -DCURL_DISABLE_TESTS=ON \
```